test/connectors/RegistrationRequestConnectorSpec.scala

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package connectors
+
+import base.SpecBase
+import com.github.tomakehurst.wiremock.client.WireMock.{created, ok, post, urlEqualTo}
+import models.requests.RegistrationRequest
+import play.api.Application
+import play.api.test.Helpers.running
+import testutils.{RegistrationData, WireMockHelper}
+import uk.gov.hmrc.http.HeaderCarrier
+
+class RegistrationRequestConnectorSpec extends SpecBase with WireMockHelper {
+
+  val registration = RegistrationData.createNewRegistration()
+
+  implicit private lazy val hc: HeaderCarrier = HeaderCarrier()
+
+  private def application: Application =
+    applicationBuilder()
+      .configure("microservice.services.one-stop-shop-registration.port" -> server.port)
+      .build()
+
+  "submitRegistration" - {
```

Comment:
Could we add tests for other cases - e.g. when the backend returns an error code or a 409?

Reply:
Will do this once we've made the changes on the backend so we no what to expect here.